### PR TITLE
➕ Add Stylelint

### PR DESCRIPTION
**Pull request by m-wiemers**
- Add Stylelint
- config stylelintry.json
- solve conflicts with eslint

https://stylelint.io/user-guide/get-started
https://github.com/prettier/stylelint-config-prettier

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,6 +7,8 @@
     "build": "next build",
     "start": "next start",
     "lint": "eslint .",
+    "stylelint": "stylelint \"**/*.css\"",
+    "stylelint-check": "stylelint-config-prettier-check",
```
Comment (lmachens) [inline package.json]: Not sure if this script is useful.

**Review by lmachens — APPROVED** [review]
🤘

**Review by christophvogel — APPROVED** [review]
GJ 👍

**Review by SilviaBarcellini — APPROVED** [review]
lgtm 👍

File: package.json (hunk 1 above)
Comment (m-wiemers) [reply package.json]: #metoo

**Pull request merged**